scripts/ubuntu_build_setup.sh

```diff
@@ -37,7 +37,9 @@ function install_release_dependencies {
 function install_runtime_dependencies {
   echo "Install runtime dependencies"
   # xvfb is needed in order run squish test into a headless server 
-  apt install -yq libxcomposite-dev xvfb libxft-dev
+  apt install -yq libxcomposite-dev xvfb libxft-dev \
+      libxcb-randr0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 \
+      libxcb-render-util0 libxcb-render0 libxcb-xinerama0
```

Comment:
These are the xcb build dependencies for Qt5.15 (Found [here]( https://wiki.qt.io/Building_Qt_5_from_Git)):
```sudo apt-get install '^libxcb.*-dev' libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev```

Looks like we're still missing a package from these.

Reply:
We don't need "*-dev" packages at runtime